### DISCUSSION TITLE
Theme switching on page load.

### DISCUSSION THREAD

**Discussion by KrychuTM** [mantinedev/mantine#1094] [Q&A] [answered]
I've realize that the documentation always flashes the light theme then switches to the dark theme. Is that just a problem with the documentation site or is the how the themes work in Mantine itself? The theme variable is stored as dark but why does the page flash the light theme before setting it to dark? Maybe bug? I don't know I just find it a bit confusing. I've only just began reading about Mantine so I haven't exactly used it yet so I just wanted to know if that's the case.

PS: Pretty cool looking library btw. You guys are doing an amazing job.

**Comment by rtivital** [comment, marked as answer]
Documentation website is prerendered with Gatsby. If you want to avoid color scheme flashes, you should store color scheme in cookie like described in the dark theme guide – https://mantine.dev/theming/dark-theme/#save-color-scheme-in-cookie

**Reply by KrychuTM** [reply]
Ah okay was just wondering. Thanks for the response.

**Reply by ilhamgepe** [reply]
hi, how you fix the flickering when dark mode? 
The docs provided no longer exist https://mantine.dev/theming/dark-theme/#save-color-scheme-in-cookie
i try useLocalStrogae but i cant change dynamic value for default value

**Reply by rtivital** [reply]
https://mantine.dev/guides/dark-theme/#save-color-scheme-in-cookie

**Reply by ilhamgepe** [reply]
I swear I've read darkmode docs. Shame on me. Sorry wasting your time. I love mantine great work. Thanks sir ♥️♥️

**Reply by viancamaulion** [reply]
Hi! [page](https://mantine.dev/guides/dark-theme/#save-color-scheme-in-cookie) isn't found anymore. Is there an updated link on this? Thank you so much!

**Reply by rtivital** [reply]
https://mantine.dev/theming/color-schemes/

**Reply by Xymanek** [reply]
> https://mantine.dev/theming/color-schemes/

The page (no longer?) mentions anything about cookies :( 

For more context: I'm currently trying to use Mantine with Remix and being able to know the theme during SSR seems to be a missing piece.
